dev/TeachingTip/TeachingTip.cpp

```diff
@@ -817,6 +845,11 @@ void TeachingTip::OnIsLightDismissEnabledChanged()
     UpdateButtonsState();
 }
 
+void TeachingTip::OnShouldConstrainToRootBoundsChanged()
+{
+    m_createNewPopupOnOpen = true;
+}
```

Comment:
Can you add a comment about why this is? When modifying state like this away from the logic it impacts, it's helpful to have a comment explaining what it's for.